workflow/scripts/postprocess_go_enrichment.py

```
@@ -1,6 +1,9 @@
 # import pandas as pd
 from ast import literal_eval as make_tuple
 import polars as pl
+import sys
+
+sys.stderr = open(snakemake.log[0], "w")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)